apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -659,6 +660,23 @@ export const getLatestEntityRootedSubgraph: ImpureGraphFunction<
   });
 };
 
+export const modifyEntityAuthorizationRelationships: ImpureGraphFunction<
+  {
+    operation: ModifyRelationshipOperation;
+    relationship: EntityAuthorizationRelationship;
+  }[],
+  Promise<void>
+> = async ({ graphApi }, { actorId }, params) => {
+  await graphApi.modifyEntityAuthorizationRelationships(
+    actorId,
+    params.map(({ operation, relationship }) => ({
+      operation,
+      object: relationship.object.id,
```

Comment:
Let's call this `resource`, agreed it is marginally less mental effort, and less vague if it ends up deconstructed and separated from `subject`

Reply:
Renamed `object` to `resource`: [`d836690` (#3423)](https://github.com/hashintel/hash/pull/3423/commits/d836690e8ab320d7c1037ffd89bd25c33df4b9bf)